src/Runner.Worker/JobContext.cs

```diff
@@ -56,5 +56,31 @@ public DictionaryContextData Container
                 }
             }
         }
+
+        public double? CheckRunID
```

Comment:
nit: `CheckRunId` per csharp naming conventions

Reply:
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalizing-compound-words-and-common-terms